docs/D-Modularity/functions.md

```diff
@@ -45,7 +45,7 @@ The module named `hello.c` starts executing at statement `int main(void)`, outpu
 
 We can sub-divide a programming project in different ways.  We select our modules so that each one focuses on a narrower aspect of the project.  Our objective is to define a set of modules that simplifies the complexity of the original problem. 
 
-![](https://ict.senecacollege.ca//~ipc144/pages/images/modules.png)
+![modules](https://ict.senecacollege.ca//~ipc144/pages/images/modules.png)
```

Comment:
It's hard to make out the arrows in this image on dark mode. The background the arrows lie on is transparent, maybe we can replace that with a solid white like this?

![modules](https://user-images.githubusercontent.com/86167097/140444265-ddb7a9d8-3920-45b9-9982-803c22ba57be.png)

Reply:
Yup I noticed this, and I can make the change tomorrow or Saturday if you have an idea how to solve it :)

Reply:
I think I solved it @a-rokay. Took a sec but I added a span for the img. It's global but it seems to make the images look good in light mode. The span uses a class to set background-color to #ffffff

Let me know if it looks good 💯

Reply:
Looks great, nice solution.